pkgs/misc/cups/drivers/fxlinuxprintutil/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchzip, dpkg, autoPatchelfHook, cups, tcl, tk, xorg, makeWrapper }:
+let
+  debPlatform =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "amd64"
+    else if stdenv.hostPlatform.system == "i686-linux" then "i386"
+         else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+in
+stdenv.mkDerivation rec {
+  name = "fxlinuxprintutil-${version}";
+  version = "1.1.1-1";
```

Comment:
where does this version even come from?

Reply:
it’s the version that the vendor has given the Debian and RPM packages

```
% curl -O --user-agent Mozilla/5.0 https://onlinesupport.fujixerox.com/driver_downloads/fxlinuxpdf112119031.zip
% unzip fxlinuxpdf112119031.zip 
Archive:  fxlinuxpdf112119031.zip
   creating: fxlinuxpdf112119031/
  inflating: fxlinuxpdf112119031/fxlinuxprint-1.1.2-1.i386.rpm  
  inflating: fxlinuxpdf112119031/fxlinuxprint-1.1.2-1.x86_64.rpm  
  inflating: fxlinuxpdf112119031/fxlinuxprintutil-1.1.1-1.i386.rpm  
  inflating: fxlinuxpdf112119031/fxlinuxprintutil-1.1.1-1.x86_64.rpm  
  inflating: fxlinuxpdf112119031/fxlinuxprintutil_1.1.1-1_amd64.deb  
  inflating: fxlinuxpdf112119031/fxlinuxprintutil_1.1.1-1_i386.deb  
  inflating: fxlinuxpdf112119031/fxlinuxprint_1.1.2-1_amd64.deb  
  inflating: fxlinuxpdf112119031/fxlinuxprint_1.1.2-1_i386.deb  
  inflating: fxlinuxpdf112119031/Readme.txt  
```